plugins/admission/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.14.3 AS build-env
+WORKDIR /go/src/github.com/fairwindsops/insights-plugins/
+
+
+ENV GO111MODULE=on
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+ENV GOARCH=amd64
+
+COPY opa opa
+
+WORKDIR /go/src/github.com/fairwindsops/insights-plugins/admission
+
+COPY ./admission/go.* ./
+RUN go mod download
+
+RUN go get -u github.com/gobuffalo/packr/v2/packr2
+
+COPY ./admission/pkg pkg
+COPY ./admission/cmd cmd
+RUN packr2 build /go/pkg/mod/github.com/fairwindsops/polaris*/main.go
```

Comment:
whats with the `*` here?

Reply:
This is definitely a little hacky. But go get appends a name to the directory based on the version. And running packr on just the admission controller doesn't process the dependencies.